<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeStyleSettingDlg</class>
 <widget class="QDialog" name="NodeStyleSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Node Style</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>270</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>10</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Circle Style Setting</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Color :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="circleBrushSettingBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set ...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Pen Outline</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Color :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="circlePenSettingBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set ...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Width :</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="circlePenWidthSetting">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="circlePen">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>50</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="circleBrush">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>10</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection</string>
   </property>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Line Color</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Width :</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="connectionLineWidthSetting">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Color :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectionLineSettingBtn">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set ...</string>
     </property>
    </widget>
    <widget class="QFrame" name="linePen">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>201</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Text Style Setting</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Font :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="textFontSettingBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set ...</string>
    </property>
   </widget>
   <widget class="QFrame" name="textBrush">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>150</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Text Color :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="textBrushSettingBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set ...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NodeStyleSettingDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NodeStyleSettingDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
